atividade03/src/test/java/br/edu/uepb/eda/atividade03/Test0_Fila.java

```diff
@@ -4,27 +4,57 @@
 import org.junit.Test;
 import static org.junit.Assert.*;
 
-
 public class Test0_Fila {
-	
-	static Fila_IF f ;
-		
-	@Before //Instanciando a Fila antes de cada teste
-	public void instanciaFila() throws Exception{
-		f = new FilaComPilhas();
-	}
-	
-	@Test
-	public void testVazia() throws Exception{
-		f.enqueue(Integer.MAX_VALUE);
-		assertFalse(f.isEmpty());
-	}
-	
-	@Test (expected = Exception.class)
-	public void testUnderFlow() throws Exception{
-		f.dequeue();
-	}
-	
-	//Outros testes ...
 
+    static Fila_IF f;
+
+    @Before //Instanciando a Fila antes de cada teste
+    public void instanciaFila() throws Exception {
+        f = new FilaComPilhas();
+    }
+
+    @Test
+    public void testVazia() throws Exception {
+        f.enqueue(Integer.MAX_VALUE);
+        assertFalse(f.isEmpty());
+    }
+
+    @Test(expected = Exception.class)
+    public void testUnderFlow() throws Exception {
+        f.dequeue();
+    }
+
+    //Outros testes ...
+    
+    @Test
+    public void testEnqueueAndDequeue() throws Exception {
+        f.enqueue(10);
+        f.enqueue(20);
+
+        assertEquals(10, f.dequeue());
```

Comment:
man, basicamente o junit ele se perde na hora de validar pois o tipo `int` é diferente de `Integer` ai a gente precisa informar pra ele um valor de desvio, [nesse link](https://stackoverflow.com/questions/41893633/ambiguous-method-call-both-assertequalsobject-object-in-assert-and-assertequa#:~:text=assertEquals(2.5%2C%20person.getScore()%2C%200.0)%3B) tem a aplicação com mais detalhes

```suggestion
        assertEquals(10, f.dequeue(), 0);
```

ai basicamente tem que fazer isso em todos os assertEquals que estão dando erro